### PR TITLE
Style the login logo with a css class

### PR DESCRIPTION
**Pull request by cproensa**
Fixes: #23382

### CODE REVIEW

File: core/layout_api.php [hunk 1]
```diff
@@ -1250,3 +1250,11 @@ function layout_scroll_up_button() {
 	echo '<i class="ace-icon fa fa-angle-double-up icon-only bigger-110"></i>' . "\n";
 	echo '</a>' . "\n";
 }
+
+function layout_login_page_logo() {
+	?>
+	<div class="login-logo">
+		<img src="<?php echo helper_mantis_url( config_get( 'logo_image' ) ); ?>">
+	</div>
```
Comment (dregad) [inline core/layout_api.php]: I'm not sure why an `<h1>` tag was used before, but is there any particular reason to change from that to a `<div>` ?

Comment (cproensa) [reply core/layout_api.php]: > but is there any particular reason to change from that to a <div>?

div is a neutral tag that does not carry on any formatting, and is better suited to apply a css customization to change the positioning outside of the container flow, if needed.

Comment (dregad) [reply core/layout_api.php]: Makes sense.

File: core/layout_api.php [hunk 2]
```diff
@@ -1250,3 +1250,11 @@ function layout_scroll_up_button() {
 	echo '<i class="ace-icon fa fa-angle-double-up icon-only bigger-110"></i>' . "\n";
 	echo '</a>' . "\n";
 }
+
+function layout_login_page_logo() {
```
Comment (atrol) [inline core/layout_api.php]: Missing PHPDoc block

**Comment by cproensa** [comment]
note. It must be done also on signup, verify, lost-pwd, etc

**Comment by atrol** [comment]
Closed via https://github.com/mantisbt/mantisbt/commit/883655a761cf9aa35fc82b311487d06a6cd9088e

**Pull request closed without merging**